### PR TITLE
Fixes #58

### LINKED ISSUE #58: Pull helper methods out of src/main/java/com/sendgrid/smtpapi/SMTPAPI.java
There are quite a few helper methods in `SMTPAPI`  - pull them out and put them in another location where these helper methods can be reused.

Example: 
```
private static String[] toArray(JSONArray json) {
    ArrayList<String> parse = new ArrayList<String>();
    for (int i = 0; i < json.length(); i++) {
      parse.add(json.getString(i));
    }
    return parse.toArray(new String[parse.size()]);
  }
```

https://codeclimate.com/github/sendgrid/smtpapi-java/src/main/java/com/sendgrid/smtpapi/SMTPAPI.java#issue_59f5623652fc6c000100001c

### PR DESCRIPTION
**Pull request by huytranrjc**
….java

# Fixes #58

### Checklist
- [ ] I have made a material change to the repo (functionality, testing, spelling, grammar)
- [ ] I have read the [Contribution Guide] and my PR follows them.
- [ ] I updated my branch with the master branch.
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation about the functionality in the appropriate .md file
- [ ] I have added in line documentation to the code I modified

### Short description of what this PR does:
- 
- 

If you have questions, please send an email to [Sendgrid](mailto:dx@sendgrid.com), or file a Github Issue in this repository.

### CODE REVIEW

**Comment by SendGridDX** [comment]
[CLA assistant check](https://cla.sendgrid.com/sendgrid/smtpapi-java?pullRequest=59) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.sendgrid.com/sendgrid/smtpapi-java?pullRequest=59) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.sendgrid.com/check/sendgrid/smtpapi-java?pullRequest=59) it.</sub>

**Comment by mbernier** [comment]
@huytranrjc there was an error on build with `JSONArray`, can you please look into this?

```
The TaskContainer.add() method has been deprecated and is scheduled to be removed in Gradle 2.0. Please use the create() method instead.
:compileJavawarning: [options] bootstrap class path not set in conjunction with -source 1.6
/home/travis/build/sendgrid/smtpapi-java/src/main/java/com/sendgrid/helpers/Helpers.java:5: error: modifier static not allowed here
public static class Helpers {
              ^
/home/travis/build/sendgrid/smtpapi-java/src/main/java/com/sendgrid/smtpapi/SMTPAPI.java:10: error: package com.sendgrid.smtpapi.helpers does not exist
import com.sendgrid.smtpapi.helpers.Helpers;
                                   ^
/home/travis/build/sendgrid/smtpapi-java/src/main/java/com/sendgrid/helpers/Helpers.java:7: error: cannot find symbol
  public static String[] toArray(JSONArray json) {
                                 ^
  symbol:   class JSONArray
  location: class Helpers
3 errors
1 warning
 FAILED
```

File: src/main/java/com/sendgrid/helpers/Helpers.java [hunk 1]
```diff
@@ -0,0 +1,43 @@
+package com.sendgrid.smtpapi;
+
+import java.util.ArrayList;
+
+public static class Helpers {
+
+  public static String[] toArray(JSONArray json) {
```
Comment (mbernier) [inline src/main/java/com/sendgrid/helpers/Helpers.java]: This is where the error I reported in comments showed up

**Comment by thinkingserious** [comment]
Since there has been no activity on this issue since March 1, 2020, we are closing this issue. If you are still interested in moving this PR forward, please feel free to reopen. Thank you!

**Pull request closed without merging**